src/foam/u2/search/FilterController.js

```diff
@@ -336,28 +332,6 @@ foam.CLASS({
       return filterView;
     },
 
-    function buildFilter(args) {
-      var e = this.Card.create({ padding: false });
-      e.style({
-        'margin-bottom': '0',
-        overflow: 'visible'
-      }).cssClass(this.myCls('filter-container'))
-          .start('div')
-              .cssClass(this.myCls('filter-header'))
-              .add(args.label)
-          .end()
-          .startContext({ data: args.key })
-          .add(args.showRemove ? this.REMOVE_FILTER : undefined)
-          .endContext()
-      .end();
-
-      e.start('div')
-          .cssClass(this.myCls('filter-body'))
-          .add(args.view)
-      .end();
-      return e;
-    },
-
     function renderFilter(key) {
       this.filtersE_.add(this.searchViews_[key]);
```

Comment:
Can we make this more functional?

We should add this.filtersE$ and make filtersE an expression of something.